data/talkactions/scripts/start_cast.lua

```diff
@@ -0,0 +1,9 @@
+function onSay(cid, words, param)
+	local player = Player(cid)
```

Comment:
This is really pedantic, but we can safely change cid to player and remove local player = Player(cid) for both start_cast.lua and stop_cast.lua